### PR TITLE
Copied .travis.yml from source given below

### PR DESCRIPTION
**Pull request by ferenckata**
## Description
Setting up CI with arduino and android
https://github.com/DIT112-V19/arduino-android-ci/blob/master/.travis.yml

## Related issue(s)
Issue #27 and #46

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    # Install Travis CI configuration for Arduino sketches
+    - source <(curl -SLs https://raw.githubusercontent.com/adafruit/travis-ci-arduino/master/install.sh)
+
+install:
+    # Install Smartcar shield library (used by our sketch)
+    - arduino --install-library "Smartcar shield"
+
+script:
+    # Build the Android app
+    - cd $TRAVIS_BUILD_DIR/android
```
Comment (platisd) [inline .travis.yml]: I don't see a directory called `android` in your repository, so this step will definitely fail. What happens if you replace it with `carMobileApp` instead? :+1:

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    - ./gradlew build --stacktrace
+    # Run the Android app's unit tests
+    - ./gradlew test --info
+    # Run the Android app's instrumented tests
+    - ./gradlew connectedAndroidTest
+    # Build all .ino files for Arduino Mega
+    - cd $TRAVIS_BUILD_DIR/arduino
```
Comment (platisd) [inline .travis.yml]: There's also no `arduino` directory either. You'll need to adjust that accordingly.

Comment (ferenckata) [reply .travis.yml]: thank you! I already saw the failure, it's great to know how to fix

Comment (platisd) [reply .travis.yml]: Just a note, I didn't necessarily mean to rename the folders to what the script needs but instead change the script according to the folders names you have. :stuck_out_tongue:

Comment (ferenckata) [reply .travis.yml]: I think it would be cleaner to have the folders named such, but for the time being I must agree changing the code is easier

Comment (platisd) [reply .travis.yml]: Sure, no problem by me either way :+1:

**Comment by ferenckata** [comment]
already merged as part of android app test branch

**Pull request closed without merging**